### DISCUSSION TITLE
unimplemented information

### DISCUSSION THREAD

**Discussion by bundgaard** [Vector35/binaryninja-api#3070] [Analysis] [answered]
Hi

Reading a DLL for a synthesizer and the the high level IL returns code like 
```
    bool c_3 = unimplemented  {or al, byte [rax]}
```

Can I do something to make it more readable or is it backend code that needs to be updated?

**Comment by galenbwill** [comment]
What version on Binary Ninja are you using?

i believe this is fixed on current dev, but it is definitely broken on stable.

**Reply by bundgaard** [reply]
I do believe I was using the current version at the time, if you guys have released a newer version I haven't looked into it.
Sadly busy doing my money job :-/

**Comment by psifertex** [comment, marked as answer]
Also, to ask the general question -- lifting for all architectures can be changed by third-parties as the architectures are open source:

https://github.com/vector35/?q=arch-&type=all&language=&sort=

In this particular case the commit in question is: https://github.com/Vector35/arch-x86/commit/6bb510eacede26cec0b8fdcf974ec0db957cc8ad
